src/FastRandom.cpp

```diff
@@ -13,109 +13,49 @@
 #elif defined (_MSC_VER)
 	#define ATTRIBUTE_TLS static __declspec(thread)
 #else
-	#error "Unknown thread local storage qualifier"
+	#define ATTRIBUTE_TLS thread_local
```

Comment:
Seems like msvc on appveyor is failing with `thread_local`.  It has been standard since C++11 but that compiler is fairly old.